### PR TITLE
Master march 26 added course info

### PR DESCRIPTION
**Pull request by brominger**
I wrote to classes(CourseInfo and CourseData).  The Course info creates a course object with Course Number, Course Name, and Subject. The CourseData class reads the csv files and puts all of the CourseInfo into an ArrayList. I also created the Junit test for each method in the class. I might also add a section number to the CourseInfo and auto generate it when the Couse object is created.

### CODE REVIEW

File: code/src/org/txstate/auto_scheduler/CourseInfo.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package org.txstate.auto_scheduler;
+
+/**
+ * Represents a course for the school
+ * @author Barry Rominger
+ */
+
+public class CourseInfo {
+    private String course_number;
```
Comment (samaniehsan) [inline code/src/org/txstate/auto_scheduler/CourseInfo.java]: please use simple camelized names for fields like `courseNumber`

File: code/src/org/txstate/auto_scheduler/CourseInfo.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    public String getCourse_number() {
+        return course_number;
+    }
+
+    /**
+     * Changes the course number
+     * @param course_number
+     */
+
+    public void setCourse_number(String course_number) {
```
Comment (samaniehsan) [inline code/src/org/txstate/auto_scheduler/CourseInfo.java]: same comment as above `courseNumber` as opposed to `course_number`

File: code/src/org/txstate/auto_scheduler/CourseInfo.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+    public String getCourse_name() {
+        return course_name;
+    }
+
+    /**
+     * Changes course name
+     * @param course_name
+     */
+    public void setCourse_name(String course_name) {
```
Comment (samaniehsan) [inline code/src/org/txstate/auto_scheduler/CourseInfo.java]: please use camelized Names for method names like
`setCourseName`
`getCourseName`

File: code/src/org/txstate/auto_scheduler/CourseInfo.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    public String getSubject() {
+        return subject;
+    }
+
+    /**
+     * Changes subject
+     * @param subject
+     */
+
+    public void setSubject(String subject) {
```
Comment (samaniehsan) [inline code/src/org/txstate/auto_scheduler/CourseInfo.java]: i don't think we need public methods that Set data.

better to have a constructor that sets the values. so we only have public `get` data

File: testdata/test.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+Course_Number,Course_Name,Subject
+NUMBER,Name,Subject
```
Comment (samaniehsan) [inline testdata/test.csv]: better to add empty line. github likes all text files to end with an line `\n`

File: out/production/auto-scheduler/org/txstate/auto_scheduler/MANIFEST.MF [hunk 6]
```diff
@@ -0,0 +1 @@
+Main-Class: org.txstate.auto_scheduler.Main
```
Comment (samaniehsan) [inline out/production/auto-scheduler/org/txstate/auto_scheduler/MANIFEST.MF]: not sure we are adding code file to an out folder. 
this file currently exists under `https://github.com/samaniehsan/auto-scheduler/blob/master/code/src/org/txstate/auto_scheduler/MANIFEST.MF`

**Comment by samaniehsan** [comment]
@brominger thanks for being active. see my minor comments.

**Comment by brominger** [comment]
I merge with newer branch

**Pull request closed without merging**